scripts/setup/defaults.env

```diff
@@ -20,6 +20,9 @@ REGULATOR_PATH={{ cli.paths.regulator }}
 
 # --- EXPO CONTEXT ENVs --- #
 
+# Cli
+EXPO_CLI_VERSION=3.22.3
```

Comment:
Just for clarity, these new ENVs will be synced when calling `keg config sync --confirm false`